packages/katello/katello/hostname-change.rb

```diff
@@ -140,18 +140,15 @@ def next_steps_message
       else
         # the following multi-line string isn't indented because the indents are taken literally.
 %(
-  You will have to install the new bootstrap rpm and reregister all clients and #{@plural_proxy} with subscription-manager
-  (update organization and environment arguments appropriately):
+  You will have to reregister all clients and #{@plural_proxy} with the Satellite server.
```

Comment:
We don't use Satellite in upstream, but I'm not sure if we even handle branding at all.

Reply:
Haha good point I should avoid saying Satellite in this.